Changed language Drop-Down position

gabru-md opened the pull request:
Changes the position of language drop-down, placing it on the bottom-left of the screen.

Closes https://github.com/coala/projects/issues/363

<!--
Thanks for your contribution!

Please take a quick look at those things down there. They're quite important.
Really! We wrote them for you. Yes you! With utmost care. Read them.
-->

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [X] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [X] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

yukiisbored commented:
@gabru-md How does this look on mobile devices and how does it compare to desktop? Screenshots would be nice :)

gabru-md commented:
@yukiisbored  I've added the relevant screen shots. Please have a look :)
Thanks

![s1](https://user-images.githubusercontent.com/25414121/30179499-84cec4a2-9404-11e7-8cc5-4fd473ca2e21.png)
The above image is for Desktop Browser

![s2](https://user-images.githubusercontent.com/25414121/30179515-9655d24c-9404-11e7-85f5-424b37979b1c.png)
The above image is for Mobile Browser

yukiisbored commented:
@gabru-md is there a way to fix the one on mobile so it'll be always on the left side.

gabru-md commented:
yes, let me look through it.
:+1:

gabru-md commented:
Changed Mobile View : 
![m1](https://user-images.githubusercontent.com/25414121/30249173-0ea3580c-962e-11e7-9e3d-f3daad36dc99.png)
Changed Browser View :
![w1](https://user-images.githubusercontent.com/25414121/30249172-0ea1df86-962e-11e7-8a97-ab0b93c31b2f.png)
I've made them into center. looks better and the same on both platforms :D
@yukiisbored please review :D

hemangsk commented on index.html:
```diff
@@ -44,7 +44,7 @@
       <div ng-view=""></div>
     </div>
-    <div ng-controller="LanguageController as langCtrl" style="position: fixed; left: 60px; bottom: 20px;">
+    <div ng-controller="LanguageController as langCtrl" style="position: relative; margin:50% auto;width:100px">
```
sorry for nitpicking, but it'll be better to use consistent spacing  
```
position: relative; margin:50% auto;width:100px
```
to
``` 
position: relative; margin: 50% auto; width: 100px;
```

hemangsk commented on index.html:
```diff
@@ -44,7 +44,7 @@
       </div>
```
File name missing in commit message.

hemangsk commented:
Added a few comments. Screenshots look great! nice work :) 👍

gabru-md commented:
Okay ! I'll do it :D

gabru-md commented:
@hemangsk please have a look!
thanks

hemangsk commented:
Hey @gabru-md , your commit short log doesn't match the commit guidelines
http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html#shortlog says 
it should have a tag and a short description separated by a colon (:).

yukiisbored commented:
Please squash your commits.

gabru-md commented:
@yukiisbored Please Review 
I've squashed all my commits into one.

stellargo commented on index.html:
```diff
@@ -44,7 +44,7 @@
       <div ng-view=""></div>
     </div>
-    <div ng-controller="LanguageController as langCtrl" style="position: fixed; left: 60px; bottom: 20px;">
+    <div ng-controller="LanguageController as langCtrl" style="position: relative; margin: 50% auto; width: 100px">
```
semicolon needed after 100px i guess :P

soumya-ranjan7 commented:
ack 8d376be

hemangsk commented:
@rultor merge

rultor commented:
> @rultor merge

@hemangsk OK, I'll try to merge now. You can check the progress of the merge [here](http://www.rultor.com/t/13196-338440451)

rultor commented:
> @rultor merge

@gabru-md @hemangsk Oops, I failed. You can see the full log [here](http://www.rultor.com/t/13196-338440451) (spent 2min)

```
+ gpg --no-tty --batch --verbose --decrypt --passphrase rultor-key:coala/projects rultor_secrets.sh.enc
gpg: AES encrypted data
gpg: gpg-agent is not available in this session
gpg: encrypted with 1 passphrase
gpg: original file name='rultor_secrets.sh'
+ rm -rf rultor_secrets.sh.enc
+ rm -rf .gpg
+ cd repo
+ git remote add fork git@github.com:gabru-md/projects.git
+ git remote update
Fetching origin
Fetching fork
From github.com:gabru-md/projects
 * [new branch]      Adrianz/cib -> fork/Adrianz/cib
 * [new branch]      Makman2/eli -> fork/Makman2/eli
 * [new branch]      add-me     -> fork/add-me
 * [new branch]      add_alex   -> fork/add_alex
 * [new branch]      adtac/maint-check -> fork/adtac/maint-check
 * [new branch]      adtac/optcach -> fork/adtac/optcach
 * [new branch]      adtac/rultor-maint-check -> fork/adtac/rultor-maint-check
 * [new branch]      change     -> fork/change
 * [new branch]      fneu/theses -> fork/fneu/theses
 * [new branch]      gci-use-coala-tasks -> fork/gci-use-coala-tasks
 * [new branch]      genericbears -> fork/genericbears
 * [new branch]      hemangsk/docs -> fork/hemangsk/docs
 * [new branch]      hemangsk/yaml -> fork/hemangsk/yaml
 * [new branch]      master     -> fork/master
 * [new branch]      meet/report -> fork/meet/report
 * [new branch]      projects/editor_support -> fork/projects/editor_support
 * [new branch]      projects/optimize_caching -> fork/projects/optimize_caching
 * [new branch]      sanketdg/docparse -> fork/sanketdg/docparse
 * [new branch]      sils-patch-1 -> fork/sils-patch-1
 * [new branch]      sils/uhafner -> fork/sils/uhafner
 * [new branch]      sims/deadline -> fork/sims/deadline
 * [new branch]      sims/vulture -> fork/sims/vulture
 * [new branch]      ud/antlr   -> fork/ud/antlr
 * [new branch]      ud/projects -> fork/ud/projects
+ args=
+ '[' only == default ']'
+ '[' only == no ']'
+ '[' only == only ']'
+ args=' --ff-only'
+ export BRANCH=__rultor
+ BRANCH=__rultor
++ git show-branch __rultor
++ wc -l
+ '[' 0 -gt 0 ']'
+ git checkout -B __rultor fork/master
Switched to a new branch '__rultor'
Branch __rultor set up to track remote branch master from fork.
+ git checkout -B master origin/master
Switched to and reset branch 'master'
Branch master set up to track remote branch master from origin.
Your branch is up-to-date with 'origin/master'.
+ '[' false == true ']'
+ '[' false == true ']'
+ git merge --ff-only __rultor
fatal: Not possible to fast-forward, aborting.
'cid' file is absent, container wasn't started correctly

```

hemangsk commented:
@gitmate-bot rebase

hemangsk commented:
ack 1fb3903

yukiisbored commented:
@rultor merge

rultor commented:
> @rultor merge

@yukiisbored OK, I'll try to merge now. You can check the progress of the merge [here](http://www.rultor.com/t/13196-338675824)

rultor commented:
> @rultor merge

@yukiisbored Done! FYI, the full log is [here](http://www.rultor.com/t/13196-338675824) (took me 2min)
